src/main/java/com/onlinebookstore/store/model/Book.java

```diff
@@ -0,0 +1,27 @@
+package com.onlinebookstore.store.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false,unique = true)
```

Comment:
```suggestion
    @Column(nullable = false, unique = true)
```
don't forget to Ctrl Alt L autoformat